aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/exceptions/handlers/SelectiveExceptionHandler.java

```diff
@@ -25,10 +29,14 @@
  * falls back to {@code PermissiveExceptionHandler} behavior
  */
 class SelectiveExceptionHandler extends PermissiveExceptionHandler {
-    private List<String> criticalExceptions;
+    private List<String> exceptionTokens;
+    private static final String ALL_EXCEPTIONS = "all";
+    private static final String EXCEPTIONS_WILDCARD = "*";
+    private static final String EXCLAMATION_MARK = "!";
```

Comment:
Let us call this "negation sign" or "inverson sign". 
This is exactly what clean code guidelines warn against: a constant name must describe **what it does** / **what it is for** and not *what it looks like*. Constant names must explain author's intention. Or else we will get like `private static final A = "A"`